index.js

```diff
@@ -28,6 +28,9 @@ var colors = {
   brightBlack: '#747369'
 }
 
+var process = require('process') // excluded from browser
+var isNode = !!process.version
```

Comment:
Actually, we still need to use `process.env`, so we need to leave `require('process')` so we can exclude it from the browser build using the package.json "browser" field.

If we use the `process` global instead of `require('process')` then it's not possible to exclude it.